libr/core/cmd_print.c

```diff
@@ -4847,7 +4848,7 @@ static int cmd_print(void *data, const char *input) {
 				char *s = malloc (core->blocksize + 1);
```

Comment:
use calloc, and remove the memset

Reply:
left this `malloc` to optimize redundant memory clear